go-controller/hybrid-overlay/pkg/controller/master.go

```diff
@@ -289,13 +290,87 @@ func (m *MasterController) handleOverlayPort(node *kapi.Node, annotator kube.Ann
 		}
 	}
 
+	// create a reroute on the ovn_cluster_router
+	err = createLogicalRouterPolicyForNode(node, subnets)
+	if err != nil {
+		return fmt.Errorf("failed to create logical router policy: %v", err)
+	}
+
+	return nil
+}
+
+func createLogicalRouterPolicyForNode(node *kapi.Node, subnets []*net.IPNet) error {
+	// get all the windows subnets as the dest for lr-policy match
+	var dest string
+	var nextHop string
+	var src string
+	for _, hybridSubnet := range config.HybridOverlay.ClusterSubnets {
```

Comment:
This might be linked to @trozet 's question below (https://github.com/ovn-org/ovn-kubernetes/pull/1744/commits/f6187014af8957cb1d328f30b4b791266608f0aa#r508044900), but this is not the same as what you've removed in `addRoutesGatewayIP`, that code also had a check on ` && !hasRoutingExternalGWs && !hasPodRoutingGWs`, here: 
https://github.com/ovn-org/ovn-kubernetes/blob/master/go-controller/pkg/ovn/pods.go#L216
This patch now adds the logical router policies regardless of that. Do we want that?